CHANGELOG.md

```diff
@@ -26,7 +26,7 @@
 
 ## Fixed Issues
 
--
+- [core] Fix TypeScript error, when `moment()` is used in filtering an OData field of type `Edm.DateTimeOffset`
```

Comment:
```suggestion
- [core] Fix a type error, when `moment()` is used in filtering an OData field of type `Edm.DateTimeOffset`
```
As javascript project will also face this compile time error, I'll not mention `TypeScript`.